### PR TITLE
Improve our database to allow using other engines (PostgreSQL)

### PR DESCRIPTION
**Pull request by javiereguiluz**
This PR finishes the work started by @lyrixx in #260.

### CODE REVIEW

**Comment by lyrixx** [comment]
:+1:

File: src/AppBundle/Entity/Comment.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 /**
  * @ORM\Entity
+ * @ORM\Table(name="symfony_demo_comment")
```
Comment (voronkovich) [inline src/AppBundle/Entity/Comment.php]: What about to use a table prefix for this? http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/cookbook/sql-table-prefixes.html
Or is it too advanced feature?

Comment (javiereguiluz) [inline src/AppBundle/Entity/Comment.php]: @voronkovich I didn't know about this. Thanks for sharing that link.

I don't know if others will agree, but I think that solution is too advanced even for experienced developers.

**Comment by voronkovich** [comment]
@javiereguiluz,  you forgot to update the test database's schema

File: src/AppBundle/Entity/Comment.php (hunk 1 above)
Comment (stof) [inline src/AppBundle/Entity/Comment.php]: I agree with @javiereguiluz

**Pull request merged**